Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -422,7 +423,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 }
 
                 $fixEvent = $this->stopwatch->getEvent('fixFiles');
-                $output->writeln(sprintf('Fixed all files in %.3f seconds, %.3f MB memory used', $fixEvent->getDuration() / 1000, $fixEvent->getMemory() / 1024 / 1024));
+                $output->writeln(sprintf('Processed all files in %.3f seconds, %.3f MB memory used', $fixEvent->getDuration() / 1000, $fixEvent->getMemory() / 1024 / 1024));
+                if (!$showProgress) {
```

Comment:
My thoughts were that it will duplicate the status info. Actually, i did that in the very first implementation, but later realized that it's probably too much. But I don't mind to show the both results, if needed.